src/ignitus-Api/index.js

```diff
@@ -25,12 +25,12 @@ export async function getTestimonialData() {
   }
 }
 
-export function signUp(email, password) {
-  return axios.post(t.STUDENT_SIGN_UP, { email, password, userType: 'student' });
+export function signUp(email, password, userType) {
+  return axios.post(t.STUDENT_SIGN_UP, { email, password, userType });
```

Comment:
reverting this change as signUp is still not implemented for prof, so it has affected the existing functionality , moreover I have migrated auth to emotion feel to have a look :)